modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/objects/ReconciliationAffectedEntries.java

```diff
@@ -352,6 +380,6 @@ public int skippedCachesCount() {
      * @return Skipped entries count.
      */
     public int skippedEntriesCount() {
-        return skippedEntries.size();
+        return skippedEntries.values().stream().flatMap(m -> m.values().stream()).mapToInt(Set::size).sum();
```

Comment:
Seems that we miss tests for this.

Reply:
I extended my test to check this (compact and regular modes). Looks like there is only one case when `skipped` entries may appear - lost partitions.

Reply:
Seems that ttl per entry is also the reason, is that correct?

Reply:
Yes, it can be a reason for skipping an entry, but it is not implemented https://ggsystems.atlassian.net/browse/GG-26267
I only found this simple check: https://github.com/gridgain/gridgain/blob/master/modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/processor/PartitionReconciliationProcessor.java#L186